server/models/Notification.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  type: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  message: {
+    type: String,
+    required: true,
+  },
+  isRead: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  date: {
```

Comment:
You can consider the usage of mongoose timestamps.

Ref: https://masteringjs.io/tutorials/mongoose/timestamps

Reply:
used timestamps